### ISSUE TITLE
Null Activity

### ISSUE THREAD

**Issue by tommartensen** [bptlab/chimera#112]
### TL;DR: 
Null Activity (comp. silent transitions in Petri nets) to change data object states or between catching events without further logic, but automatic execution.

### Motivation
When implementing Business Processes, you may come across activities that only implement data object state changes or that are used between two catching events, as they may not be followed directly.
In the motivational example, this concerns the activities Login user and Logout User. 
![image](https://user-images.githubusercontent.com/9576848/41822086-9b01d942-77ea-11e8-9b75-0c07f621ae5a.png)

### Proposal
So far, the Webservice Task Activity was misused to implement this behaviour. 
We propose a Null Activity (following silent transitions in Petri nets) to replace the empty Webservice Task Activity. This activity would be used in any case, where two catching events would follow each other directly, or a data object must be written into another state without any data attribute changes.
The implementation would be done in both *Gryphon* and *Chimera*.

**Comment by julianweise** [comment]
As given in our above modeled example and perfectly described by @tommartensen, it would be great to have this feature implemented.
@marhew What do you think about it? Is there any other solution we did not come across yet? I think we should be able to implement it.

**Comment by julianweise** [comment]
As a little side note: Misusing service tasks does not only contradicts its original purpose but also makes it harder to efficiently validate configurations / inputs for service tasks.

**Comment by marhew** [comment]
This is a good idea, although I don't like the name "NullActivity". Implementation should be pretty straightforward. Question is, which type of tasks in Gryphon would you use for this "EmptyActivity"?

**Comment by tommartensen** [comment]
We will think about the naming of the activity and introduce a new activity in Gryphon, of course.

**Comment by tommartensen** [comment]
Solved in #116 and https://github.com/bptlab/gryphon/pull/37
